charms/docker/dockeropts.py

```diff
@@ -2,8 +2,8 @@
 
 
 class DockerOpts:
-
-    ''' DockerOptsManager - A Python class for managing the DEFAULT docker
+    '''
```

Comment:
nit: From what I understand about Python docstrings, they start RIGHT after the third tick, so this puts a new line character as the first character. I read up on this, and I understand that the previous way was "more correct". You don't need to change it, but just keep that in mind.